crates/noirc_evaluator/src/ssa_refactor/acir_gen/mod.rs

```diff
@@ -99,6 +107,22 @@ impl Context {
                 assert_eq!(result_ids.len(), 1, "Binary ops have a single result");
                 self.ssa_value_to_acir_var.insert(result_ids[0], result_acir_var);
             }
+            Instruction::Call { func, arguments: _ } => {
+                match &dfg[*func] {
+                    Value::Function(id) => {
+                        let func = &ssa.functions[id];
+                        match func.runtime() {
+                            RuntimeType::Normal => unreachable!(),
```

Comment:
```suggestion
                            RuntimeType::Normal => unimplemented!(),
```
A normal runtime type is still reachable, builtins notably would still remain as calls in the IR.